src/univariate/continuous/chisq.jl

```diff
@@ -1,5 +1,8 @@
 """
-    Chisq(ν)
+    Chisq(;ν)
+    Chisq(;nu)
+    Chisq(;dof)
```

Comment:
`dof` isn't defined in the docstring. `nu` should also be mentioned. Same elsewhere.

Reply:
yes, docstrings still need to be changed.